install_config/router/default_haproxy_router.adoc

```diff
@@ -907,6 +907,195 @@ This example uses `curl` with a local resolver to simulate the DNS lookup:
 # curl -k --resolve anyname.example.test:443:$routerip https://anyname.example.test/
 ----
 
+[NOTE]
+====
+For routers that allow wildcard routes (`ROUTER_ALLOW_WILDCARD_ROUTES`
+set to `true`), there are some caveats to the ownership of a subdomain
+associated with a wildcard route.
+
+Prior to wildcard routes, ownership was based on the claims made for a
+host name with the oldest route winning against any other claimants.
+E.g. route `r1` in namespace `ns1` with a claim for `one.example.test`
+     would win over another route `r2` in namespace `ns2` for the same
+     host name `one.example.test` if route `r1` was older than route `r2`.
+
+In addition, routes in other namespaces were allowed to claim
+non-overlapping hostnames.
+E.g. route `rone` in namespace `ns1` could claim `www.example.test` and
+   another route `rtwo` in namespace `d2` could claim `c3po.example.test`).
+
+This is still the case if there are _NO_ wildcard routes claiming that same
+subdomain (`*.example.test` in the above example).
+
+A wildcard route will however need to claim all the host names within a
+subdomain (`*.example.test` in the above example). A wildcard route's claim
+is allowed or denied based on whether or not the oldest route for that
+subdomain (`example.test`) is in the same namespace as the wildcard route.
+The oldest route in turn can be either a regular route or a wildcard route.
+E.g. if the oldest route `eldest` in namespace `ns1` making a claim for
+     subdomain `example.test` is a route which claims hostname
+     `owner.example.test`, then only a route in the `ns1` namespace will
+     be allowed to claim a wildcard route for that subdomain (`*.example.test`).
```

Comment:
and that should be route `wildcard` not eldest probably to be clearer - sounds a bit odd on a second read.